tests/operators/test_python.py

```diff
@@ -856,6 +886,25 @@ def f():
 
         self._run_as_operator(f, python_version=3, use_dill=False, requirements=['dill'])
 
+    @staticmethod
+    def _initiate_pypi_server():
+        try:
+            if not os.path.exists('/root/packages'):
+                os.makedirs('/root/packages')
+            subprocess.Popen(['pypi-server'])
+            time.sleep(1.5)
```

Comment:
How do we know the magic value of 1.5 is good ? Are you sure it is enough for all cases (running in CI etc?). That's a very bad idea to sleep in unit tests. It slows down the whole test suite. There should be a better way to check if the server is running. One option is to run an http request to it in a loop until it succeeds if there is no better way.